.github/workflows/CI.yml

```diff
@@ -231,3 +231,50 @@ jobs:
             cd scipy/
             micromamba activate scipy-dev
             python dev.py test -t scipy.interpolate -v
+
+  misc:
+    name: Misc Test ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["macos-latest", "ubuntu-latest"]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: environment.yml
+          create-args: >-
+            lfortran=${{ env.LFORTRAN_VERSION }}
+            llvmdev=11.1.0
+
+      - uses: hendrikmuhs/ccache-action@main
+        with:
+          variant: sccache
+          key: ${{ github.job }}-${{ matrix.os }}
+
+      - name: Print installed packages information
+        shell: bash -e -x -l {0}
+        run: |
+            which lfortran
+            micromamba env list
+            micromamba activate lf
+            micromamba list
+            which lfortran
+
+      - name: Clone LFortran Source Repository
+        shell: bash -e -x -l {0}
+        run: |
+            git clone https://github.com/lfortran/lfortran.git
+            cd lfortran
+            git checkout 925797482746846763531d5d5ea2cf9995359b63
```

Comment:
For now I simply used checking out the specific commit. It seems this might not work, since the reference outputs generated by the `lfortran` `v0.33.03` and the reference outputs expected at this commit `925797482746846763531d5d5ea2cf9995359b63 ` would not be the same (the ref tests might have got updated by the time of this commit).